models/base/scratch/base_scratch.yml

```diff
@@ -38,7 +38,7 @@ models:
           - not_null
       - name: page_view_id
         description: '{{ doc("col_page_view_id") }}'
-      - name: domain_sessionid
+      - name: session_id
```

Comment:
Happy to change that, but just so that I understand, should I change `snowplow__events` everywhere to `snowplow__event_table`? The variable is defined in `dbt_project.yml`, [here](https://github.com/snowplow/dbt-snowplow-media-player/blob/issue/new_media_schemas/dbt_project.yml#L31). The same one is used in the ecommerce model, I don't see the `snowplow__event_table` used anywhere, why should we use that?

Reply:
Only here, it should be: `identifier: "{{ var('snowplow__event_table', 'events') if project_name != 'snowplow_media_player_integration_tests' else 'snowplow_media_player_events_stg' }}"` This should only provide the event table reference (when referred to it as source("atomic", "this bit")) but if you add the var `snowplow__events`, by default it refers to "itself" and compiles that bit as text due to that to something like mydatabase.atomic.{{ source("atomic", "events") }}.

Reply:
In the ecommerce model that part is also the table itself: 
<img width="860" alt="image" src="https://github.com/snowplow/dbt-snowplow-media-player/assets/95634439/3089e6c1-5c0d-4858-a683-093fcbfa165d">

Reply:
For context this is a weird dependency chain and not really how dbt wants you to work. Basically we want to define the source but then have the user be able to edit the schema and table name in case it's not `atomic.events`. We _should_ then be able to just use the source in all places, however we need the `snowplow_events` variable, which is just set to the source and the user should never change for the integration tests to work...

Reply:
I see, thanks! I have used `snowplow__events_table` to match the ecommerce package.